src/main/java/com/redhat/cloud/notifications/NotificationsApp.java

```diff
@@ -5,17 +5,36 @@
 
 import javax.enterprise.event.Observes;
 
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 public class NotificationsApp {
+
     private static final String BUILD_COMMIT_ENV_NAME = "OPENSHIFT_BUILD_COMMIT";
     private static final String BUILD_REFERENCE_ENV_NAME = "OPENSHIFT_BUILD_REFERENCE";
     private static final String BUILD_NAME_ENV_NAME = "OPENSHIFT_BUILD_NAME";
 
+    public static final String FILTER_REGEX = ".*(/health|/metrics) HTTP/[0-9].[0-9]\\\" 200.*";
+    private final Pattern pattern = Pattern.compile(FILTER_REGEX);
```

Comment:
Because `NotificationsApp` uses `@Observes StartupEvent`, Quarkus automatically transforms it to a `@Dependent` CDI bean which is eligible for garbage collection after the event has been fired.

Using an instance attribute instead of a `static` one will prevent the garbage collection so this would be better:
```suggestion
    private static final Pattern pattern = Pattern.compile(FILTER_REGEX);
```

Reply:
This hint is interesting. Thanks :) I didn't know that :)